guacamole/Dockerfile

```diff
@@ -1,13 +1,29 @@
 # SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
 # SPDX-License-Identifier: Apache-2.0
 
+FROM debian:bookworm-slim as build-extension
+USER root
+
+RUN apt-get update && apt-get install -y zip
+COPY ext /tmp/ext
+
+WORKDIR /tmp/ext
+RUN zip -r /tmp/guacamole-capellacollab.jar .
+
 FROM guacamole/guacamole:1.5.2
 
 USER root
```

Comment:
Do we still need this `USER root` here as we have a new one in line 5?

Reply:
Line 5 runs in another build context. It doesn't apply here.